src/Agent/NewRelic/Agent/Core/Configuration/DefaultConfiguration.cs

```diff
@@ -1138,6 +1138,39 @@ public TimeSpan ErrorEventsHarvestCycle
         public virtual uint ErrorsMaximumPerPeriod { get { return 20; } }
         public virtual IEnumerable<string> ExceptionsToIgnore { get { return ServerOverrides(_serverConfiguration.RpmConfig.ErrorCollectorErrorsToIgnore, _localConfiguration.errorCollector.ignoreErrors.exception); } }
 
+        public virtual IEnumerable<string> ExpectedClasses { get { return ServerOverrides(_serverConfiguration.RpmConfig.ErrorCollectorExpectedClasses, _localConfiguration.errorCollector.expectedClasses.errorClass); } }
```

Comment:
Should we consider combining this property with the ExpectedMessages property so that our error handling logic only needs to look at one of the them? This is just something to think about, and we'll see which approach we prefer when we work on using these config values.

Reply:
We should also cache this logic.

Reply:
I think we might be able to combine them together.